projects/batfish-common-protocol/src/main/java/org/batfish/common/plugin/IBatfish.java

```diff
@@ -156,7 +156,9 @@ AnswerElement standard(
       String ingressNodeRegexStr,
       String notIngressNodeRegexStr,
       String finalNodeRegexStr,
-      String notFinalNodeRegexStr);
+      String notFinalNodeRegexStr,
+      Set<String> transitNodes,
+      Set<String> notTransitNodes);
```

Comment:
If these are allowed to be `null`, annotate them `@Nullable`. But honestly, seems easier if they're not and we just require the caller to pass the empty set.